### ISSUE TITLE
Cannot import CSS file

### ISSUE THREAD

**Issue by hummingly** [omgovich/react-colorful#90]
**Version**: ^4
**Bundler**: webpack@^5.4.0
**Loader**: css-loader@^4.3.0

Hello :)
I use the distributed CSS file and import it like it is explained in the example but webpack cannot find the file. I've pinpointed the problem to the recent package.json changes. The exports field misses the CSS file and adding it fixed my compilation error.

```
"exports": {
    ".": {
      "browser": "./dist/index.module.js",
      "umd": "./dist/index.umd.js",
      "import": "./dist/index.module.js",
      "require": "./dist/index.js"
    },
    "./dist/index.css": "./dist/index.css"
}
```

**Comment by omgovich** [comment]
Hi! Just tested — your suggestion fixes the problem.

Webpack 5 is a new version and I guess they started to prefer `exports` map when it's present.
Will release a new version of our package ASAP.

**Comment by hummingly** [comment]
Thanks for the quick response!

**Comment by omgovich** [comment]
Just published `4.4.2`. Should be good now.
Thanks for your issue! Let me know if you need anything else 🍻
